lkml/__init__.py

```diff
@@ -96,6 +96,27 @@ def parse_args(args: Sequence) -> argparse.Namespace:
         help="increase logging verbosity to debug",
     )
 
+    group = parser.add_mutually_exclusive_group()
+    group.add_argument(
+        "--json",
+        action="store_true",
+        default=True,
+        help="return a JSON string (default)",
+    )
+    group.add_argument(
+        "--lookml",
+        action="store_true",
+        default=False,
+        help="return a LookML string",
+    )
+    group.add_argument(
+        "-w",
+        "--write",
+        action="store_true",
+        default=False,
+        help="parse and write back to file",
+    )
```

Comment:
What's the use case for parsing from a file and writing back to the same file?

Reply:
The lkml parser/dumper does a great job at cleaning up lkml files (separating measures from dimensions, correcting the indentation, etc.), the option to write it back to file would allow to use `lkml` as a auto-formatter in text editors, pre-commits, CI checks, etc. (like running `black` on .py files).